packages/cashier/src/pages/withdrawal/withdrawal-locked/withdrawal-locked.tsx

```diff
@@ -21,7 +21,7 @@ const WithdrawalLocked = observer(() => {
         is_10k_withdrawal_limit_reached: is_10K_limit,
         error: { is_ask_financial_risk_approval },
     } = withdraw;
-    const { document, identity, needs_verification } = account_status.authentication || {};
+    const { document, identity, needs_verification } = account_status?.authentication || {};
```

Comment:
In new packages, typically its better pattern to ensure that all data is complete first, and then just use the data without worrying about ?.
As ultimately, this add way more test cases and way more potential bugs and more complexity to control and less predictability (no way that one can predict how all combinations of .? will work together and what side effects will be there :D )